common/achievements/sheep-starer.ts

```diff
@@ -0,0 +1,57 @@
+import ZedaphPlaysCommon from '../cards/hermits/zedaphplays-common'
+import ZedaphPlaysRare from '../cards/hermits/zedaphplays-rare'
+import {CardComponent} from '../components'
+import query from '../components/query'
+import {CardEntity} from '../entities'
+import {onTurnEnd} from '../types/priorities'
+import {achievement} from './defaults'
+import {Achievement} from './types'
+
+const zedaphCards = [ZedaphPlaysCommon, ZedaphPlaysRare]
+
+const SheepStarer: Achievement = {
+	...achievement,
+	numericId: 14,
+	id: 'sheep_starer',
+	name: 'Sheep Starer',
+	icon: '',
+	description: 'Keep a red HP zedaph alive for three consecutive turns.',
```

Comment:
```suggestion
	description: 'Keep a red HP (100 - 0) Zedaph alive for three consecutive turns.',
```
I think it would be a good idea to put the numbers in